File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] n, int target) {
        int l=0;
        int r=n.length-1;
        while(l<r){
            int sum=n[l]+n[r];
            if(sum>target){
                r--;
            }else if(sum<target){
                l++;
            }else{
                return new int[]{l+1,r+1};
            }
        }
        return new int[]{-1,-1};
    }
}

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution(object):
    def twoSum(self, n, target):
        """
        :type numbers: List[int]
        :type target: int
        :rtype: List[int]
        """
        i=0
        j=len(n)-1
        while(i<j):
            s=n[i]+n[j]
            if(s==target):
                return [i+1,j+1]
            elif(s>target):
                j-=1
            else:
                i+=1
        
